rasa_sdk/knowledge_base/storage.py

```diff
@@ -0,0 +1,136 @@
+import logging
+import random
+
+
+logger = logging.getLogger(__name__)
+
+
+SCHEMA_KEYS_KEY = "key"
+SCHEMA_KEYS_ATTRIBUTES = "attributes"
+SCHEMA_KEYS_REPRESENTATION = "representation"
+MANDATORY_SCHEMA_KEYS = [
+    SCHEMA_KEYS_KEY,
+    SCHEMA_KEYS_ATTRIBUTES,
+    SCHEMA_KEYS_REPRESENTATION,
+]
+
+
+class KnowledgeBase(object):
+    def __init__(self, schema):
+        self.ordinal_mention_mapping = {
+            "one": lambda l: l[0],
```

Comment:
isn't there a library for that?

Reply:
also duckling should be able to extract that and pass it to us, right?

Reply:
yes, but adding duckling service as a dependency for such a simple thing would be overkill IMO

Reply:
as @Ghostvv  suggested, could we just use synonyms for this mapping? so `the [first](mention:1) one?` then the `ordinal_mention_mapping` dict can be made language agnostic.

Reply:
Yes, good idea. Did that already with the `entity_type` and `attributes`.